rita/src/rita_common/traffic_watcher/mod.rs

```diff
@@ -112,11 +112,8 @@ pub fn watch<T: Read + Write>(
         // Only ip6
         if let IpNetwork::V6(ref ip) = route.prefix {
             // Only host addresses and installed routes
-            if ip.get_netmask() == 128 && route.installed {
-                destinations.insert(
-                    IpAddr::V6(ip.get_network_address()),
-                    Int256::from(route.price + local_price),
-                );
+            if ip.prefix() == 128 && route.installed {
```

Comment:
does this mean larger prefixies won't get billed?

Reply:
yes, because right now because we assume one address = one node = a /128 route